Makefile

```diff
@@ -829,37 +823,45 @@ clippy-microvm:
 # Rules for Running System Level Tests Using Nanvix Daemon
 #===================================================================================================
 
+# List of supported functions in hyperlight.
+HYPERLIGHT_WHITELIST := echo-c echo-cpp echo-rust-nostd hello-c hello-cpp
```

Comment:
@ppenna it was not very clear to me what tests can actually run in `hyperlight`. I was checking a succesful CI run, and observed that: (1) none of the linuxd tests were running for hyperlight, and (2) from nanvixd only a handful: https://github.com/nanvix/nanvix/actions/runs/16472768859/job/46566063191#step:5:3059.

I have surfaced this with this explicit allow list. Please let me know if any other tests need to be added here.